apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -100,23 +106,31 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
     struct Vote {
         uint256 yea;                                        // Voting power for
         uint256 nay;                                        // Voting power against
+        // Note: rename to totalPower?
         uint256 votingPower;                                // Total voting power (based on the snapshot block)
         uint256 settingId;                                  // Identification number of the setting applicable to the vote
         uint256 actionId;                                   // Identification number of the associated disputable action on the attached Agreement
+        // Note: we may want to re-order this state to be based on localized SSTOREs
+        //       start date and snapshot block never change, whereas pausedAt, pauseDuration and status may be changed together
         VoteStatus status;                                  // Status of the vote
         uint64 startDate;                                   // Datetime when the vote was created
         uint64 snapshotBlock;                               // Block number used to check voting power on attached token
         uint64 pausedAt;                                    // Datetime when the vote was paused
         uint64 pauseDuration;                               // Duration of the pause (only updated once resumed)
+        // Note: maybe rename to quietEndingExtensionDuration?
         uint64 quietEndingExtendedSeconds;                  // Total number of seconds a vote was extended due to quiet ending
+        // Note: I initially found this name a bit confusing, but it makes sense now
         VoterState quietEndingSnapshotSupport;              // Snapshot of the vote's support at the beginning of the first quiet ending period
+        // Note: let's prefer to store a hash here
         bytes executionScript;                              // EVM script attached to the vote
         mapping (address => VoteCast) castVotes;            // Mapping of voter address => more information about their cast vote
     }
 
+    // Note: may be good to double check if we'd like these to be updatable and push them to the settings
```

Comment:
Something to double check with @lkngtn; being able to change the time could be useful for the future.

Even if we don't let it be changed, it is something we could store in the settings array already (just a bit worried about gas cost, but I think this flexibility could be worth it for the 2/3k extra gas on votes).

Reply:
Confirmed offline we do want to introduce this change, addressed in 97b53e6f9231b36c828ce1c4e9c8de0d1fce2596